src/pages/login/Login.tsx

```diff
@@ -0,0 +1,71 @@
+import React, { FC } from "react";
+import { Button, Col, Icon, Row } from "@canonical/react-components";
+import { Navigate, useNavigate } from "react-router-dom";
+import { useAuth } from "context/auth";
+import Loader from "components/Loader";
+import BaseLayout from "components/BaseLayout";
+import { useSettings } from "context/useSettings";
+
+const Login: FC = () => {
+  const { isAuthenticated, isAuthLoading } = useAuth();
+  const navigate = useNavigate();
+  const { data: settings } = useSettings();
+  const hasOidc = settings?.auth_methods?.includes("oidc");
+
+  if (isAuthLoading) {
+    return <Loader />;
+  }
+
+  if (isAuthenticated) {
+    return <Navigate to="/ui" replace={true} />;
+  }
+
+  return (
+    <BaseLayout>
+      <Row className="empty-state">
+        <Col size={6} className="col-start-large-4">
+          <Icon name="containers" className="empty-state-icon lxd-icon" />
+          <h1 className="p-heading--4 u-sv-2">LXD Login</h1>
```

Comment:
Is the "LXD" prefix needed? Seems implicit to me.
```suggestion
          <h1 className="p-heading--4 u-sv-2">Login</h1>
```